locales/nl.yml

```diff
@@ -0,0 +1,256 @@
+home.open_device_failed: Apparaat openen mislukt
+home.open_device_failed_linux: Apparaat openen mislukt, raadpleeg hoofdstuk 1.3 van de gebruikershandeling
+
+# Main Window, General
+kbd.general: Algemeen
+kbd.general.info: Informatie
+kbd.general.info.device_model: Apparaat Model
+kbd.general.info.serial_number: Serienummer
+kbd.general.info.battery: Batterij
+kbd.general.info.running_time: Looptijd
+kbd.general.setting: Setting
+kbd.general.setting.layout: Layout
+kbd.general.setting.tap_layout: Tik
+kbd.general.setting.magic_fn: Magic Fn
+kbd.general.setting.macro_trigger: Macro
+kbd.general.setting.capslock_led: CapsLock Led
+kbd.general.setting.auto_sleep: Automatische slaapstand
+kbd.general.setting.options.disable: Uitschakelen
+kbd.general.setting.options.enable: Inschakelen
+kbd.general.setting.options.default: Standaard
+kbd.general.setting.options.custom: Custom
+kbd.general.setting.options.fn1: Fn1
+kbd.general.setting.options.fn2: Fn2
+kbd.general.setting.options.capslock_all: All
+kbd.general.bluetooth: Bluetooth
+kbd.general.bluetooth.bind: Bluetooth binden
+kbd.general.bluetooth.clean_bind: Clean binden
+
+# Main Window, Firmware
+kbd.firmware: Firmware upgrade
+kbd.firmware.key: Applicatieprocessor
+kbd.firmware.led: Lichtprocessor
+kbd.firmware.ble: Bluetoothprocessor
+kbd.firmware.list: Firmwarelijst
+kbd.firmware.annepro1.download.firmware: Firmware downloaden
+kbd.firmware.annepro1.download.latest_firmware: Download laatste firmware
+kbd.firmware.annepro1.download.dfu_tool: Download DFU Tool
+kbd.firmware.find_new_version: 'Vind nieuwe firmware: '
+kbd.firmware.no_new_version: De huidige firmware is de laatste.
+kbd.fireware.upgrade.btn: Upgraden
+kbd.firmware.upgrade.btn.to_latest: Upgrade naar nieuwste
+kbd.firmware.upgrade.title: Firmware upgrade
+kbd.firmware.upgrade.content: Werp de USB-kabel NIET uit tijdens het upgraden
+kbd.firmware.upgrade.status.downloading: Firmware aan het downloaden...
+kbd.firmware.upgrade.status.download_failed: Firmware downloaden mislukt
+kbd.firmware.upgrade.status.entering_aip: Upgrademodus...
+kbd.firmware.upgrade.status.writing_key: Data aan het schrijven naar de applicatieprocessor...
+kbd.firmware.upgrade.status.writing_led: Data aan het schrijven naar de lichtprocessor...
+kbd.firmware.upgrade.status.writing_ble: Data aan het schrijven naar de bluetoothprocessor...
+kbd.firmware.upgrade.status.restart: Toetsenbord opnieuw op aan het starten...
+kbd.firmware.upgrade.status.success: Firmware upgraden gelukt.
+
+# Main Window, LEDs
+kbd.led: Light
+kbd.led.effect.group: Light Group
+kbd.led.effect.group.reset: Reset
+kbd.led.effect.group.delete.confirm_content: Weet je zeker dat je deze lichteffectgroep wilt verwijderen?
+kbd.led.effect.list: Lichtlijst
+kbd.led.effect.list.limit.annepro1_three: Je kan alleen maar 3 lichteffecten per 1 groep hebben 
+kbd.led.effect.list.limit.annepro2_sixteen: Je kan alleen maar 6 lichteffecten per 1 groep hebben

```

Comment:
Here should be `kbd.led.effect.list.limit.annepro2_sixteen: Je kan alleen maar 16 lichteffecten per 1 groep hebben`

Reply:
Ah, right, didn't notice. Thanks, I will fix it.